### DISCUSSION TITLE
No response from GitHub Support for disabled github actions

### DISCUSSION THREAD

**Discussion by amanPras** [community/community#45331] [Actions]
### Select Topic Area

General

### Body

Even after 13 days, I've received no response to my GitHub Support Ticket for disabled GitHub actions.
Is this normal? or is there a different mechanism to handle the such request?

Looked into https://github.com/orgs/community/discussions/23989. I share the same pain but couldn't see any solution there.

**Comment by amanPras** [comment]
Can I get some help, please

**Comment by ghost** [comment]
me too, bro
